src/main/java/de/tum/in/www1/artemis/config/MetricsBean.java

```diff
@@ -89,6 +88,8 @@ public class MetricsBean {
 
     private final StatisticsRepository statisticsRepository;
 
+    private List<String> cachedActiveUserNames;
```

Comment:
Is there any particular reason why this is not a Set?

Reply:
No, it could also be a `Set` (or any other `Collection`).
Would you prefer a `Set` since it clarifies that the usernames are unique?

Reply:
that was one aspect, I was thinking about. I also thought that sets are more efficient but this is apparently only the case if it's a many-to-many-relationship. Therefore, I think we can leave it as it is.

Reply:
Okay, then I would leave it as it is.